src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -64,7 +64,12 @@ public class OtlpExporterOptions
         /// <summary>
         /// Gets or sets optional headers for the connection.
         /// </summary>
-        public Metadata Headers { get; set; } = new Metadata();
+        public string Headers { get; set; }
+
+        /// <summary>
+        /// Gets or sets the max waiting time (in milliseconds) for the backend to process each span.
+        /// </summary>
+        public int? TimeOut { get; set; }
```

Comment:
I think elsewhere in the project we use the one-word "timeout" so "Timeout" here I think would be more consistent. I prefer to include the units too, so my vote would be: `public int? TimeoutInMilliseconds { get; set; }` or TimeoutMilliseconds is fine too, used other spots.

Reply:
+1 on [`Timeout`](https://en.wikipedia.org/wiki/Timeout_(computing))

Reply:
`TimeoutMilliseconds` will be consistent with other part of this repo.

Reply:
Fixed this.